artifacts/ubuntu/ubuntu.go

```diff
@@ -37,6 +37,7 @@ func (u *ubuntu) Metadata() *api.Metadata {
 			Username: "ubuntu",
 		},
 		EnvVariables: u.EnvVariables,
+		Arch:         architecture.GetImageArchitecture(u.Arch),
```

Comment:
Can you move normalization of `u.Arch` to a single place instead? E.g. in `retrieveArchitectureArtifact`? (applies to all artifacts)